src/rollback_to_stable/rts_btree.c

```diff
@@ -48,14 +48,14 @@ __rts_btree_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *firs
         if (!txn_id_visible || rollback_timestamp < upd->durable_ts ||
           upd->prepare_state == WT_PREPARE_INPROGRESS) {
             __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-              "rollback to stable update aborted with txnid: %" PRIu64
-              ", txnid not visible: %s, or stable timestamp (%s) < durable timestamp (%s): %s, or "
-              "prepare state (%d) is in progress: %s",
+              "[UPDATE_ABORT] rollback to stable update aborted with txnid=%" PRIu64
```

Comment:
Could we use macros to define these common prefix strings? It would make it less likely that a typo causes an annoying diagnostic exercise. The code should be able to look very similar, it would just become:

```
#define RTS_VERB_TAG_UPDATE_ABORT "[RTS_UPDATE_ABORT]"
    RTS_VERB_TAG_UPDATE_ABORT "rollback to stable update aborted with"
```

Since string concatenation in C will magic those together for you. See usages of WT_COMPAT_MSG_PREFIX for an example.

Reply:
Thanks - I've changed this in 4735e9b, let me know what you think.